src/Evolution/ComputeTags.hpp

```diff
@@ -11,81 +15,50 @@
 #include "Domain/Tags.hpp"
 #include "PointwiseFunctions/AnalyticSolutions/Tags.hpp"
 #include "Time/Tags.hpp"
+#include "Utilities/Gsl.hpp"
 #include "Utilities/TMPL.hpp"
 
-/// \cond
-namespace gsl {
-template <typename T>
-class not_null;
-}  // namespace gsl
-/// \endcond
-
-namespace evolution {
-namespace Tags {
+namespace evolution::Tags {
 /*!
- * \brief Use the `AnalyticSolutionTag` to compute the analytic solution of the
- * tags in `AnalyticFieldsTagList`.
+ * \brief Computes the analytic solution and adds `::Tags::Analytic` of the
+ * `std::optional<Tensor>`s to the DataBox.
  */
-template <size_t Dim, typename AnalyticSolutionTag,
-          typename AnalyticFieldsTagList>
-struct AnalyticCompute : ::Tags::AnalyticSolutions<AnalyticFieldsTagList>,
-                         db::ComputeTag {
+template <size_t Dim, typename AnalyticFieldsTagList>
+struct AnalyticSolutionsCompute
```

Comment:
I'd like to avoid a rename to minimize subtle conflicts in the next series of PRs... We probably need to restructure how we do evolution tags more broadly anyway

Reply:
Ok 👍